lib/project_types/script/messages/messages.rb

```diff
@@ -113,6 +113,10 @@ module Messages
           dependency_install_cause: "Something went wrong while installing the needed dependencies.",
           dependency_install_help: "Correct the errors.",
 
+          invalid_environment_cause: "Your environment %{tool} version, %{env_version}, "\
+                                     "is too low. It must be greater than %{minimum_version}.",
+          invalid_environment_help: "Update your %{tool}",
```

Comment:
I agree with Jimmy about the period at the end. 
Instead of "Something went wrong...." could we say:
"Your script wasn't created because your environment %{tool} version (%{env_version}) "\
                                     "is too low. It must be greater than %{minimum_version}.",